client-side/src/stories/RadioButton/radio-Button.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Radio, FormControlLabel, RadioGroup, FormControl } from '@mui/material';
+import './radio-Button.scss';
+
+const RadioButton = ({ options, selectedOption, onChange }) => {
```

Comment:
you should use memo, because the radio buttons are rendered and the options aren't changed.
put all this function inside memo()